client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -149,7 +150,14 @@ const ManageAllDomainsButton = styled( Button )`
 const SitesDashboardSitesList = createSitesListComponent();
 
 export function SitesDashboard( {
-	queryParams: { page = 1, perPage = 96, search, status = 'all', newSiteID },
+	queryParams: {
+		page = 1,
+		perPage = 96,
+		search,
+		status = 'all',
+		isFilterByOwner = false,
```

Comment:
What do you think about simplifying the new query param?

Reply:
I changed the parameter to `owner="me"` https://github.com/Automattic/wp-calypso/pull/84257/commits/65ab7cd03c27281e6d6adc190f976bdeb56f3eca

Reply:
Thanks, works well and looks great now.